docs/source/getting_started.rst

```diff
@@ -139,10 +85,36 @@ The complete state of the Ankaios system can be retrieved using the `get_state`
 The complete state contains information regarding the workloads running in the Ankaios cluster, configurations and agents. The state can be filtered using filter masks
 (See `get_state <ankaios.html#ankaios_sdk.ankaios.Ankaios.get_state>`_).
 
+**Update a workload**
+---------------------
+
+Considering we have the above workload running, we can now modify it. For this example we will update the ``restartPolicy``. To be able to pinpoint
+the exact workload we want to modify, we must know only it's name. 
+
+.. code-block:: python
+
+    from ankaios import Ankaios
+
+    # Create an Ankaios object
+    ankaios = Ankaios()
+
+    # Get the workload based on the name
+    workload = ankaios.get_workload("nginx")
```

Comment:
Can we also rename the workload in the example manifest to `nginx` to reduce confusion.